crates/ruff_python_formatter/src/comments/visitor.rs

```diff
@@ -336,7 +339,7 @@ fn text_position(comment_range: TextRange, source_code: SourceCode) -> CommentLi
 ///
 /// Used by [`CommentStyle::place_comment`] to determine if this should become a [leading](self#leading-comments), [dangling](self#dangling-comments), or [trailing](self#trailing-comments) comment.
 #[derive(Debug, Clone)]
-pub(super) struct DecoratedComment<'a> {
+pub(crate) struct DecoratedComment<'a> {
```

Comment:
I've been applying pub(crate) liberally after rust complained about private types in public interfaces

Reply:
I can see how this is needed but it breaks with the intended encapsulation of the module.